docs/src/_reference/command-line-interface.md

```diff
@@ -732,11 +731,69 @@ meltano job remove <job_name>
 
 ### Examples
 
+##### Tasks and Blocks
+
+A task in a job consists of one or more _blocks_. A block is simply a set of related plugins that will be executed serially.
+For example, you can create a job that consists of a single tap and target:
+
+```bash
+meltano job add tap-gitlab-to-target-postgres --tasks "tap-gitlab target-postgres"
+```
+
+This would add the following to your `meltano.yml`:
```

Comment:
Can we please update the main `meltano.yml` documentation in https://docs.meltano.com/concepts/project#meltanoyml-project-file to cover the format of the `jobs` list as well? Since people will be finding that in their team's `meltano.yml` and want to learn what it's about without knowing `meltano job add` yet.